base/scripts/objectives.cs

```diff
@@ -273,16 +283,16 @@
         }
         //OTHERWISE DO NOTHING HANDLED BY SUICIDE COUNTER NOW
     }
-    //MessageAll(0, "clientTime: " @ %curTimeLeft);
     
+    //MessageAll(0, "clientTime: " @ %curTimeLeft);
     UpdateClientTimes(%curTimeLeft);
     }
   }
 }
 
 function Game::UpdateTimeOnly() {
     
-    %curTimeLeft = ($Server::timeLimit * 60) + $missionStartTime - $simTimeTemp;
+    %curTimeLeft = ($Server::timeLimit * 60) + ($missionStartTime - $simTimeTemp);
```

Comment:
note - this doesn't do anything

5 + (4 - 3) is the same as 5 + 4 - 3

Reply:
true but my life is more at ease when things are in parenthesis.